e2e/libs/backup/backup.py

```
@@ -41,6 +43,17 @@ def verify_no_error(self, volume_name):
         assert not backup_volume['messages'], \
             f"expect backup volume {volume_name} has no error, but it's {backup_volume['messages']}"
 
+    def verify_errors(self):
+        for i in range(self.retry_count):
+            logging(f"Waiting for error backups ... ({i})")
+            backups = self.backup.list_all()["items"]
+            for backup in backups:
+                if backup["status"]["state"] == "Error":
+                    logging(f"Got error backup {backup}")
+                    return
+            time.sleep(self.retry_interval)
```

Ruff (0.8.2)

Undefined name `time`

(F821)